.github/workflows/staging-release.yaml

```diff
@@ -969,13 +980,20 @@ jobs:
           token: ${{ secrets.GH_PA_TOKEN }}
           ref: '4.0'
 
-      - name: Release 4.1 and latest
+      - name: Release 4.1 - not latest
         if: startsWith(inputs.version, '4.1')
         uses: actions/checkout@v5
         with:
           repository: fluent/fluent-bit-docs
           token: ${{ secrets.GH_PA_TOKEN }}
```

Comment:
@edsiper / @cosmo0920 need to add `ref: 4.1` here otherwise it'll checkout master branch